libs/governance/src/lib/proposals-hooks/use-proposal-event.ts

```diff
@@ -28,24 +28,15 @@ export const useProposalEvent = (transaction: VegaTxState) => {
           variables: { partyId },
         })
         .subscribe(({ data }) => {
-          if (!data?.busEvents?.length) {
+          if (!data?.proposals) {
             return;
           }
 
-          // No types available for the subscription result
-          const matchingProposalEvent = data.busEvents.find((e) => {
-            if (e.event.__typename !== 'Proposal') {
-              return false;
-            }
+          // typo in 'proposals' - this should be singular
```

Comment:
suggestion (non blocking): core ticket for this?

Reply:
https://github.com/vegaprotocol/vega/issues/7513